### PR TITLE
Support custom guzzle client

### PR DESCRIPTION
**Pull request by joostfaassen**
Hi @malc0mn  and @VoganWong what do you think about this implementation?

It's backwards compatible, allows for custom guzzle client, and doesn't include composer.phar

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -39,6 +39,13 @@ public function __construct($server = '', $version = 'v2')
         );
     }
 
+    public static function constructWithGuzzleClient(GuzzleClient $guzzleClient)
```
Comment (malc0mn) [inline src/Client.php]: Does this method not have a missing `$server` and `$version` parameter?

**Comment by joostfaassen** [comment]
@malc0mn missing params added, thanks! does it otherwise look good to merge + release?

**Comment by malc0mn** [comment]
I personally see no further issue with it, so go ahead I'd say :)

**Comment by joostfaassen** [comment]
Great, merged and tagged v1.4.0. Thank you!

**Pull request merged**